src/patternfly/components/Avatar/styles.scss

```diff
@@ -1,19 +1,7 @@
 @import "../../patternfly-utilities";
 
-// Component variables follow this formula:
-// --pf-c-avatar__element--modifier--state--PropertyCamelCase
-
-// The value of component scoped variables is always defined by a global variable:
 .pf-c-avatar {
-  /* stylelint-disable */
-
-  // Header avatar
-  --pf-l-page__header-avatar--Width:                        pf-size-prem(48px);
-  --pf-l-page__header-avatar--Height:                       pf-size-prem(48px);
-  --pf-l-page__header-avatar--MarginLeft:                   var(--pf-global--spacer--md);
-
-  /* stylelint-enable */
+  --pf-c-avatar--BorderRadius: 50%;
```

Comment:
can't we use `--pf-global--BorderRadius--lg` here?

Reply:
doh! yes 👍